LICENSE-3RD-PARTY.txt

```diff
@@ -0,0 +1,54 @@
+-----------------------------------------------------------------------------
+                              MIT License
+  Applies to: 
+  - core-js-pure; Copyright (c) 2014-2023 Denis Pushkarev
+  - weakmap-polyfill; Copyright (c) 2015-2021 polygonplanet
+  - typedarray;
+      Copyright (c) 2010, Linden Research, Inc.
+      Copyright (c) 2012, Joshua Bell
+  - es6-promise;
+      Copyright (c) 2014 Yehuda Katz, Tom Dale, Stefan Penner and contributors
+  - colorjs.io; Copyright (c) 2021 Lea Verou, Chris Lilley
+  - emoji-regex; Copyright Mathias Bynens <https://mathiasbynens.be/>
+  - css-selector-parser; Copyright (c) 2013 Dulin Marat
+  - doT.js; Copyright (c) 2011 Laura Doktorova
+      Software includes portions from jQote2 Copyright (c) 2010 aefxx,
+      http://aefxx.com/ licensed under the MIT license.
+-----------------------------------------------------------------------------
+
+Permission is hereby granted, free of charge, to any person obtaining a copy
+of this software and associated documentation files (the "Software"), to deal
+in the Software without restriction, including without limitation the rights
+to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+copies of the Software, and to permit persons to whom the Software is
+furnished to do so, subject to the following conditions:
+
+The above copyright notice and this permission notice shall be included in
+all copies or substantial portions of the Software.
+
+THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+THE SOFTWARE.
+
+
+-----------------------------------------------------------------------------
+                              ISC License
+  Applies to: 
+  - memoizee; Copyright (c) 2012-2018, Mariusz Nowak, @medikoo, medikoo.com
```

Comment:
`memoizee` has the following transitive runtime dependencies that need to be reflected in this file:

```
d
es5-ext
es6-iterator
es6-symbol
es6-weak-map
event-emitter
ext
is-promise
lru-queue
next-tick
timers-ext
type
```